mm2src/mm2_bitcoin/spv_validation/src/helpers_validation.rs

```diff
@@ -45,8 +45,8 @@ pub enum SPVError {
     InvalidVout,
     #[display(fmt = "merkle proof connecting the `tx_id_le` to the `confirming_header`")]
     BadMerkleProof,
-    #[display(fmt = "Unable to get merkle tree from network or storage: {}", _0)]
-    UnableToGetMerkle(String),
+    #[display(fmt = "Unable to get merkle tree from network or storage: {err} for {coin}")]
+    UnableToGetMerkle { err: String, coin: String },
```

Comment:
let's display the coin before the error since the error can be a long message, so this can be
```rust
#[display(fmt = "Unable to get merkle tree from network or storage for {coin}: {err}")]
UnableToGetMerkle { coin: String, err: String },
```